shap/benchmark/__init__.py

```diff
@@ -4,13 +4,13 @@
 from . import models
 from .. import datasets
 
-from .metrics import consistency_guarantees
+# from .metrics import consistency_guarantees
 from .metrics import local_accuracy
 from .metrics import runtime
-from .metrics import remove_positive_retrain
-from .metrics import remove_negative_retrain
-from .metrics import keep_positive_retrain
-from .metrics import keep_negative_retrain
+# from .metrics import remove_positive_retrain
+# from .metrics import remove_negative_retrain
+# from .metrics import keep_positive_retrain
+# from .metrics import keep_negative_retrain
 from .metrics import remove_positive_mask
 from .metrics import remove_negative_mask
 from .metrics import keep_positive_mask
```

Comment:
I think these imports from .metrics could be put on the same line?

Reply:
is there some pep style that suggests it should be done this way?

Reply:
These are for the old API and the new framework would only depend on framework.py and perturbation.py to benchmark the explainers.

Reply:
will the old API be removed eventually?  if we keep it, maybe we should just clean up some of the existing code?

Reply:
It will be eventually. I will for now get rid of the commented code and keep the rest.